plugins/woocommerce/src/Internal/StockNotifications/Admin/NotificationsPage.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\StockNotifications\Admin;
+
+use Automattic\WooCommerce\Internal\StockNotifications\Admin\NotificationsListTable;
+use Automattic\WooCommerce\Internal\StockNotifications\Notification;
+
+/**
+ * Notifications admin page for Customer Stock Notifications.
+ */
+class NotificationsPage {
+
+	/**
+	 * Page URL.
+	 *
+	 * @const PAGE_URL
+	 */
+	const PAGE_URL = 'admin.php?page=wc-customer-stock-notifications';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+
+		// Select action.
+		$action = '';
+
+		// Nonce is checked in NotificationsPage::delete and NotificationsPage::output just displays the page.
+		if ( isset( $_GET['notification_action'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			$action = wc_clean( wp_unslash( $_GET['notification_action'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		}
+
+		switch ( $action ) {
+			case 'delete':
+				$this->delete();
+				break;
+			default:
+				$this->output();
+				break;
+		}
+	}
+
+	/**
+	 * Render page.
+	 */
+	public static function output() {
+
+		$search = isset( $_REQUEST['s'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['s'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$table  = new NotificationsListTable();
+		$table->prepare_items();
+		include __DIR__ . '/Views/html-admin-notifications.php';
```

Comment:
No need to capitalize "Views"; it can just be "views". Typically, we capitalize folders that are part of the namespace itself.